tket/src/Circuit/include/Circuit/CircPool.hpp

```diff
@@ -237,9 +237,35 @@ Circuit ZZPhase_using_TK2(Expr alpha);
 /** Equivalent to ZZPhase, using CX and Rz gates */
 Circuit ZZPhase_using_CX(Expr alpha);
 
-/** Equivalent to TK2, using CX and single-qubit gates */
+/** Equivalent to XXPhase, using ZZPhase and H gates */
+Circuit XXPhase_using_ZZPhase(Expr alpha);
+
+/** Equivalent to YYPhase, using ZZPhase and V/Vdg gates */
+Circuit YYPhase_using_ZZPhase(Expr alpha);
+
+/** Equivalents to TK2, using CX and single-qubit gates, using 1, 2 or 3 CX.
```

Comment:
Please can each function have its own doxygen comment (even if this means some copy-pasting between them).
Same for below.

Reply:
Done.